language/functional_tests/tests/testsuite/payments/print_counterfeit_money.mvir

```diff
@@ -0,0 +1,53 @@
+//! account: default, 100000
```

Comment:
This is a great example! But not sure that it's a great test because it makes it hard to see which behaviors are disallowed. Perhaps two smaller tests for doing `move_to_sender` on a generic and `move_from` on a generic would be clearer?

I definitely think we should keep this example in our back pocket to add as a test once we do support doing these on generic types.

Reply:
The ones in `generics/resource_safety` test individual behaviors. I'll add a bunch more there to cover every instruction affected.